services/user.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        email=email,

```

Comment:
why pass email field here since it's optional?

Reply:
builtin `create_user` function has optional email argument, so i decided that we don't need extra check for email